editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -899,7 +899,16 @@ Error EditorSceneImporterGLTF::_parse_meshes(GLTFState &state) {
 				array[Mesh::ARRAY_NORMAL] = _decode_accessor_as_vec3(state, a["NORMAL"], true);
 			}
 			if (a.has("TANGENT")) {
-				array[Mesh::ARRAY_TANGENT] = _decode_accessor_as_floats(state, a["TANGENT"], true);
+				PoolVector<float> tans = _decode_accessor_as_floats(state, a["TANGENT"], true);
```

Comment:
GLTF, CSG and Sprite3D weren't present in #16096, so I never have reasoned about them. I can't tell whether they are fine or not.